src/components/controls/dropdown/Dropdown.md

```diff
@@ -93,25 +119,35 @@ const options = [{
   optionValue: 'danger'
 }];
 
-<div>
-  <Dropdown
-    labelText="Success"
-    options={options}
-    validationState="success"
-  />
-
-  <Dropdown
-    labelText="Warning"
-    options={options}
-    validationState="warning"
-  />
-
-  <Dropdown
-    labelText="danger"
-    options={options}
-    validationState="danger"
-  />
-</div>
+class DropdownExample extends React.Component {
+  constructor() {
+    this.state = { selectedOption: options.find(opt => opt.optionValue === 'success') };
+
+    this.optionChanged = this.optionChanged.bind(this);
+  }
+
+  optionChanged(event) {
+    const value = event.target.value;
+    const selectedOption = options.find(opt => opt.optionValuegi === value);
```

Comment:
You've got a typo here that breaks this example (optionValuegi => optionValue).  Other than this, looks good.